Security

```diff
@@ -0,0 +1,93 @@
+Security
+OpenMRS is in contact with Security PS, an enterprise security consulting company.Security PS donated time to enhance the security of OpenMRS. Their application security experts found and privately reported software security weaknesses to the OpenMRS team. 
```

Comment:
Could you point me to the link where you found this?
Also, such information shouldn't be on a guide. Don't you think?

Reply:
ok i'll change the content